nodes/src/nodes/llm_openai_api/openai_client.py

```
@@ -0,0 +1,78 @@
+        elif isinstance(error, APIConnectionError):
+            return True
+        else:
+            return super().is_retryable_error(error)
+
+    def map_exception(self, error):
+        """
+        Convert unfriendly openai exceptions to friendlier ones.
+        """
+        if isinstance(error, AuthenticationError):
+            return ValueError('Invalid API key.')
+        elif isinstance(error, APIError):
```

Ruff (0.15.7)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)